CONTRIBUTING.md

```diff
@@ -272,6 +272,21 @@ from [the Shelley ledger spec](./eras/shelley/formal-spec)).
 4. Add a link to the package near the bottom of [flake.nix](./flake.nix),
    following the existing examples.
 
+### To update the conformance test
+
+To update the conformance test, do the following:
+
+1. Clone the [Agda specification repo](https://github.com/IntersectMBO/formal-ledger-specifications)
+2. Run `nix-build -A ledger.hsSrc` in the cloned repo, mark down the output path
+   in the nix store
+3. Clone the [executable spec repo](https://github.com/input-output-hk/cardano-ledger-executable-spec)
+4. Delete all the files in the repo and copy
```

Comment:
I find this a little confusing:
>  Delete all files in the repo and copy .. into the executable spec repo. 

Are we talking about the same repo in both cases (the same as the one cloned at the step above)? 
How about something like
4. Replace the content of the repo cloned above with the files at `/nix/store/<output of the nix-build>/haskell/Ledger/*`
```
rm -rf cardano-ledger-executable-spec/*
cp -r  /nix/store/<output of the nix-build>/haskell/Ledger/*  cardano-ledger-executable-spec
```
?

Reply:
Yeah you understood correctly. I'll change that line to what you proposed.